backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -93,6 +110,32 @@
         (response {:status (:status ed)
                    :body (:reasonPhrase ed)})))))
 
+(defn tenant-admin?
+  [request-draft api-root tenant user-id]
+  (let [admin-group-id (-> (client/get (format "%s/group-by-path/akvo/lumen/%s/admin"
+                                               api-root tenant)
+                                       request-draft)
+                           :body json/decode (get "id"))
+        admins (-> (client/get (format "%s/groups/%s/members" api-root admin-group-id)
+                               request-draft)
+                   :body json/decode)
+        admin-ids (into #{}
+                        (map #(get % "id"))
+                        (filter #(and (get % "emailVerified")
+                                      (get % "enabled"))
+                                admins))]
+    (not (empty? (set/intersection #{user-id} (set admin-ids))))))
```

Comment:
I think you could use `contains?` here as well instead of `(not (empty (set/intersection ...)))`